docs/src/modules/developing/pages/development-process-js.adoc

```diff
@@ -57,17 +57,16 @@ For more details and examples take a look at the following topics:
 [#_create_unit_tests]
 == Create unit tests
 
-It is good practice to write unit tests as you implement your services. The JavaScript kickstart codegen project packages Mocha and Chai and is configured for test-driven development. For implementation details see the https://developer.lightbend.com/docs/akka-serverless/tutorial/node-cart-test.html#_ii_add_unit_tests[Shopping cart tutorial - Create unit tests] topic.
+It is good practice to write unit tests as you implement your services. The JavaScript kickstart codegen project packages Mocha and Chai and is configured for test-driven development.
 
 [#_package_service]
 == Package service
 
 Use a Docker image to package your service and any of its dependencies. See the following pages for more information:
 
-* https://developer.lightbend.com/docs/akka-serverless/getting-started/set-up-development-env.html#_docker[Installing Docker]
-* https://developer.lightbend.com/docs/akka-serverless/deploying/docker.html[Packaging with Docker]
-* https://developer.lightbend.com/docs/akka-serverless/deploying/registries.html[Configuring registries]
-* https://developer.lightbend.com/docs/akka-serverless/tutorial/node-cart-package-deploy.html[Example of how to package a service]
+* https://developer.lightbend.com/docs/akka-serverless/setting-up/index.html#_containers[Installing Docker]
+* https://developer.lightbend.com/docs/akka-serverless/quickstart/cr-value-entity-javascript.html#_package_and_deploy_your_service[Packaging with Docker]
+* https://developer.lightbend.com/docs/akka-serverless/projects/container-registries.html[Configuring registries]
```

Comment:
@retgits in this section, links for Installing Docker and Configuring Registries were easy fixes.

Packaging though? There used to be 2 different links, both broken now. From what I can find, [this](https://developer.lightbend.com/docs/akka-serverless/quickstart/cr-value-entity-javascript.html#_package_and_deploy_your_service) is the best description of packaging a service built with the JavaScript SDK, so that's why I'm linking to it.

Reply:
We can do that for now, but it probably makes more sense to have a topic in the `services` section of the main docs that talks about this. It'll be very similar for all languages so I don't think it makes sense to have it duplicated in each SDK we create.

Reply:
Agreed. I created an Issue to capture this https://github.com/lightbend/akkaserverless-docs/issues/937